soc/xtensa/intel_adsp/common/ipc.c

```diff
@@ -88,7 +92,11 @@ int intel_adsp_ipc_init(const struct device *dev)
 		config->regs->idd = INTEL_ADSP_IPC_DONE;
 	} else {
 		config->regs->ida = INTEL_ADSP_IPC_DONE;
-		config->regs->tda = INTEL_ADSP_IPC_DONE;
+		#ifdef CONFIG_SOC_SERIES_INTEL_ACE
```

Comment:
style: please fix the style here and elsewhere, move #ifdef/#else/#endif to the start of the line and remove code indentation.